apps/betterangels/app.config.js

```diff
@@ -13,7 +13,7 @@ export default {
     name: IS_PRODUCTION ? 'BetterAngels' : 'BetterAngels (Dev)',
     slug: 'betterangels',
     scheme: IS_PRODUCTION ? 'betterangels' : 'betterangels-dev',
-    version: '1.0.20',
+    version: '1.0.21',
```

Comment:
Hi @Davit-BetterAngels 

I noticed a couple things:

1. I created a new profile, and uploaded some files. What I noticed was the date: it shows 11/09/23 on all of the files. 
Can you change this to whenever the date the user uploaded? 
2. When I tap the Download the file button, it takes about 4-5 seconds to see the menu to select the destination. Is it possible to make it faster? 
3. The folder name should be "Forms" but I see "Consent Form". Let me know I should create a ticket or this is the right PR to mention this but in the DEV-88, I see it mentioned "Forms". Can you modify this?

Everything else looks good on Download / Delete (I see some of the things that need to be fixed but I will create tickets for them.) Thanks so much!

<img width="390" src="https://github.com/user-attachments/assets/0db8be06-1b02-482e-b14d-6ff558459cfc"/>
<img width="390" src="https://github.com/user-attachments/assets/928fe9bd-9737-4bc9-baa0-58bea689431e"/>